snuba/util.py

```diff
@@ -165,8 +187,11 @@ def column_expr(dataset, column_name, body, alias=None, aggregate=None):
     if aggregate:
         expr = function_expr(aggregate, expr)
 
-    alias = escape_col(alias or column_name)
-
+    use_escape_alias = state.get_config("use_escape_alias", 0)
+    # This is a precaution during the rollout.
```

Comment:
This is the only non trivial change to the existing logic that can impact other datasets. Thus I will deploy this disabled, then once safe in prod we can quickly switch it on and off in case of issues. If everything is fine I will remove the check.

Reply:
Makes sense. Do you have anything specifically you'd expect to see if it goes bad (other than just "errors"?)

Reply:
if something goes wrong here, most certainly the query will fail because it would be badly formatted.

Reply:
Makes sense, thank you.